text/1005-edgedb-project.rst

```diff
@@ -0,0 +1,418 @@
+  that the configured instance is up-to-date.
+
+Interactive mode
+----------------
+
+Here's a simulation of a proposed interactive mode for a new project::
+
+    $ edgedb project init
+    `edgedb.toml` was not found in `/home/user/work/myapp` or above.
+    Do you want to initialize a new project? [Y/n] Y
+    What type of EdgeDB instance would you like to use with this project?
+    1. Local (native)
+    2. Local (Docker)
+    3. Cloud
+    Your choice? 1
+    Specify the version of EdgeDB to use with this project [latest stable]:
+    Specify the name of EdgeDB instance to use with this project [myapp]:
+
+    [shows summary of configuration]
+
+    [asks whether to continue or restart configuration]
+
+    Creating instance `myapp`...
+
+Here's a simulation of a proposed interactive mode for a cloned project::
+
+    $ edgedb project init
+    Found `edgedb.toml` in `/home/user/work/myapp`.
+    Found no associated EdgeDB instance.
+    What type of EdgeDB instance would you like to use with this project?
+    1. Local (native)
+    2. Local (Docker)
+    3. Cloud
+    Your choice? 1
+    Specify the name of EdgeDB instance to use with this project [myapp]:
+
+    [shows summary of configuration]
+
+    [asks whether to continue or restart configuration]
+
+    Creating instance `myapp` ...
+    Running migrations ...
+
+
+edgedb project deinit
+=====================
+
+Remove association with and optionally destroy the linked EdgeDB intstance.
+
+Synopsis
+--------
+
+``edgedb project deinit [options]``
+
+Options
+-------
+
+``--project-dir=<dir>``
+  Specifies a project root directory explicitly.  If not specified, the project
+  directory is detected as described in the "Project detection" section above.
+
+``--destroy-server-instance, -D``
+  If specified, the associated EdgeDB instance is destroyed by running
+  ``edgedb server destroy``.
+
+``--non-interactive``
+  Run in non-interactive mode.  Assume affirmative answer for all questions.
+
+Implementation
+--------------
+
+The ``edgedb project deinit`` command removes the association with its EdgeDB
+instance by removing the corresponding entry from the ``~/.edgedb/projects``
+directory.  If ``--destroy-server-instance`` is specified, the associated
+instance is destroyed.
+
+
+edgedb project status
+=====================
+
+Shows the information about a project.
+
+Synopsis
+--------
+
+``edgedb project status [options]``
+
+Options
+-------
+
+``--project-dir=<dir>``
+  Specifies a project root directory explicitly.  If not specified, the project
+  directory is detected as described in the "Project detection" section above.
+
+``--json``
+  Use JSON as output format.
+
+
+Implementation
+--------------
+
+The ``edgedb project status`` shows the following information about the
+project:
+
+- associated EdgeDB instance name, or `<none>` if not associated;
+- migration status (which deprecates ``edgedb show-status``)
```

Comment:
Why would we deprecate `show-status`? What to do with remote/cloud instances?

Reply:
`show-status` is about migrations, you are thinking about `server status` here.  That said, there is still value in the ability to query the status of migrations on an instance without an explicit project.  That said, we should still deprecate `show-status` as we've agreed to rename it to `migration-status` for clarity (see RFC 1003).